engine/src/environments/stratego_related/strategostate.h

```diff
@@ -0,0 +1,113 @@
+/*
+  CrazyAra, a deep learning chess variant engine
+  Copyright (C) 2018       Johannes Czech, Moritz Willig, Alena Beyer
+  Copyright (C) 2019-2020  Johannes Czech
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+/*
+ * @file: strategostate.h
+ * Created on 05.2021
+ * @author: BluemlJ
+ *
+ * A more specialized OpenSpiel wrapper for the Stratego Implementation which is based
+ * on OpenSpiel. 
+ */
+
+#ifndef STRATEGOSTATE_H
+#define STRATEGOSTATE_H
+
+#include "state.h"
+#include "open_spiel/spiel.h"
+#include "open_spiel/games/yorktown.h"
+
+class StateConstantsStratego : public StateConstantsInterface<StateConstantsStratego>
+{
+public:
+    static int BOARD_WIDTH() {
+        return open_spiel::yorktown::BoardSize();
+    }
+    static int BOARD_HEIGHT() {
+        return  open_spiel::yorktown::BoardSize();
+    }
+    static int NB_CHANNELS_TOTAL() {
+        return open_spiel::yorktown::InformationStateTensorShape().front();  // TODO
```

Comment:
Is the TODO information still applicable here?

Reply:
```suggestion
        return open_spiel::yorktown::InformationStateTensorShape().front(); 
```

No, it is not.